### ISSUE TITLE
JAWs skips over the `document-metadata` section 

### ISSUE THREAD

**Issue by ajkiessl** [psu-libraries/psulib_blacklight#1190]
For individual records, when JAWs passes the availability section it skips over the `document-metadata` section and goes right to the footer.  It might be a good idea to add header labels to the headers in the `document-metadata` section.
